### ISSUE TITLE
Variable Problems with Shure Axient

### ISSUE THREAD

**Issue by Sound-Techniker** [bitfocus/companion-module-shure-ulxd#30]
Hi Guys,

im struggeling with triggers and Variables.

i would like to use the var "$(AD4Q_1-4:ch_1_battery_runtime)"
to execute an Http Trigger.
in Companion contain the var "7:30" for example.
The condition i want to programm is "$(AD4Q_1-4:ch_1_battery_runtime) < 7:29"
but nothings happens.

i tested with other variables like 
## $(AD4Q_1-4:ch_1_battery_charge) 
and this was not an problem.

whats the Problem? Can me help somebody?
Thx

**Comment by MeestorX** [comment]
Did you try using the internal compare variables action? (Forget what they're called specifically)

**Comment by Sound-Techniker** [comment]
Yes i created an trigger who watch for changes in var $(AD4Q_1-4:ch_1_battery_runtime). If the change Happens the condition is to compare the variable $(AD4Q_1-4:ch_1_battery_runtime) with the value i set („7:29“) when the values are equal an trigger should fired. Because this was Not working i changed for testing the var to an other, i talked before, with charge (%) or with Bars (1-5). This worked fine. The Problem is only with Runtime so far

**Comment by krocheck** [comment]
You're trying to do a time calculation/compare with a variable that is simply a text string.  You probably want to change the variable format in the instance config to "Numeric Only". That will change the "7:30" to "450", representing minutes as an integer that you can do a proper compare to.

**Comment by Sound-Techniker** [comment]
Thank you, this was an option for me that i cant figure out because companion doesnt mark the Variabletypes. I Will try this

**Comment by Sound-Techniker** [comment]
Hi Guys so yesterday finally i had time to test the last tips.
i tested the tips from last post from "krochek".
![photo_2024-01-14_10-47-04](https://github.com/bitfocus/companion-module-shure-wireless/assets/48445427/5142a55f-3d4f-470a-b857-d8d361040743)
![photo_2024-01-14_10-46-57](https://github.com/bitfocus/companion-module-shure-wireless/assets/48445427/456e1f16-d810-4498-adc0-1e34271ba036)

in my Case now in "Numeric Only" and switched off and on again the Modul.
now i was awaiting an numerical like the "450" but it is still "7:30".

Any suggestions what i could do wrong?

**Comment by Sound-Techniker** [comment]
hey guys the problem still exists. it would be nice to have the option the raw minutes runtime as a variable.
by example : $(AD4Q_1-4_var:ch_x_battery_runtime_mins)
